tests/unit/components/DisplayTaxonName.test.js

```diff
@@ -36,106 +36,110 @@ const uncapitalizedTaxon = factory( "LocalTaxon", {
   rank_level: 10
 } );
 
-describe( "when common name is first", () => {
+describe( "when common name is first", ( ) => {
   const user = { prefers_scientific_name_first: false };
 
-  test( "renders correct taxon for species", () => {
-    render(
-      <DisplayTaxonName item={{ taxon: speciesTaxon, user }} />
-    );
+  test( "renders correct taxon for species", ( ) => {
+    render( <DisplayTaxonName item={{ taxon: speciesTaxon, user }} /> );
 
-    expect(
-      screen.getByText( `${speciesTaxon.preferred_common_name} (${speciesTaxon.name})` )
-    ).toBeTruthy();
+    expect( screen.getByTestId( "display-taxon-name" ) ).toHaveTextContent(
+      `${speciesTaxon.preferred_common_name} ${speciesTaxon.name}`
+    );
   } );
 
-  test( "renders correct taxon w/o common name", () => {
-    render(
-      <DisplayTaxonName item={{ taxon: noCommonNameTaxon, user }} />
-    );
+  test( "renders correct taxon w/o common name", ( ) => {
+    render( <DisplayTaxonName item={{ taxon: noCommonNameTaxon, user }} /> );
 
-    expect( screen.getByText( noCommonNameTaxon.name ) ).toBeTruthy();
+    expect( screen.getByTestId( "display-taxon-name" ) ).toHaveTextContent(
+      noCommonNameTaxon.name
+    );
   } );
 
-  test( "renders correct taxon w/o common name and no species", () => {
-    render(
-      <DisplayTaxonName item={{ taxon: highRankTaxon, user }} />
-    );
+  test( "renders correct taxon w/o common name and no species", ( ) => {
+    render( <DisplayTaxonName item={{ taxon: highRankTaxon, user }} /> );
 
-    expect(
-      screen.getByText( `${highRankTaxon.rank} ${highRankTaxon.name}` )
-    ).toBeTruthy();
+    expect( screen.getByTestId( "display-taxon-name" ) ).toHaveTextContent(
+      `${highRankTaxon.rank} ${highRankTaxon.name}`
+    );
   } );
 
-  test( "renders correct taxon for a subspecies", () => {
-    render(
-      <DisplayTaxonName item={{ taxon: highRankTaxon, user }} />
-    );
+  test( "renders correct taxon for a subspecies", ( ) => {
+    render( <DisplayTaxonName item={{ taxon: highRankTaxon, user }} /> );
```

Comment:
`highRankTaxon` has `rank_level: 27` in your test, which makes it a subfamily, not a subspecies, so you should change the name of this test.